google/ggrc-core
# GGRC-5658 Disable mappings of Scoping object - Standard and Scoping object - Regulation

# Issue description

* The system should not allow to create mapping between scoping object and standard / regulation on BE (blacklisted on GGRC api).
* The system should not allow to remove mapping between scoping object and standard / regulation on BE (blacklisted on GGRC api).
* The system should not allow to create mapping between scoping object and standard / regulation via import.
* The system should not allow to remove mapping between scoping object and standard / regulation via import.

# Steps to test the changes

* check mapping and unmapping between scoping object and Standard/Regulation with UI (the error message should be shown)
* check mapping and unmapping between scoping object and Standard/Regulation via import (the warning message should be shown)

# Solution description

* Validation on relationship creation was added
* Validation on relationship delete was added
* Validation was added to column handler for creation and deletion of a mapping

# Sanity checklist




# PR Review checklist
